include/NAS2D/Mixer/MixerNull.h

```diff
@@ -5,7 +5,7 @@
 namespace NAS2D
 {
 
-	class MixerNull : public Mixer
+	class MixerNull final : public Mixer

```

Comment:
I understand your reasons for marking this class `final` but carefully consider this decision. Refer to `C.139: Use final sparingly`: https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#Rh-final

Reply:
I would agree `final` shouldn't be overused. Here it seems justified, though whether or not it's necessary is another thing. I'm open to taking it out if we think it's a bad idea.

I suppose maybe we could end up writing something like a `MixerNullWithLogging`. And the potential slicing from misuse through a base class pointer seems pretty unlikely to happen, so it doesn't really protect from much.

I suspect this is the sort of thing you have better intuition about after making a mistake with it.